shared/src/business/entities/NewPetitionerUser.ts

```diff
@@ -41,40 +49,61 @@ export class NewPetitionerUser extends JoiValidationEntity {
     confirmPassword: joi.valid(joi.ref('password')).required(),
     email: JoiValidationConstants.EMAIL.lowercase()
       .required()
+      .messages({
+        '*': 'Enter a valid email address',
+        'string.max': 'Email address must be fewer than 100 characters', //todo test
```

Comment:
If we're going grammatically correct, should this be 'Email address must `have/contain` fewer than 100 characters?'